e2e/config/staging.js

```diff
@@ -0,0 +1,9 @@
+const launchUrl =
+  'http://telus-design-system-docs.s3-website-us-east-1.amazonaws.com/staging/components/index.html'
+const healthCheckUrl =
+  'http://telus-design-system-docs.s3-website-us-east-1.amazonaws.com/staging/components/index.html'
+
+module.exports = {
+  launchUrl,
+  healthCheckUrl,
+}
```

Comment:
Changing environments seems a bit out of scope for this first implementation, as we are not totally sure how we will using this yet. I suppose having the staging config makes sense as we may want to run visual checks before pushing to prod, but I'm not so sure about the production config.

It seems a bit overkill to have 3 files too, and then use a dynamic `require` to load one of them. I'd probably just have a single `config` object:

```js
const config = {
  development: { ... },
  staging: { ... },
  production: {... }
}

module.exports = config[process.env.APP_ENV || 'development']
```

Reply:
updated.